HOW_WE_USE_GITHUB.md

```diff
@@ -136,14 +138,38 @@ Once issues are deemed ready to be worked on, they will be moved to the [`conda`
+Each label has an associated description that clarifies how the label should be used. To see a label's description, find the label either in `conda/infrastructure`'s [`.github/global.yml` file][labels-global] or, if it is not there, in the relevant repository's `.github/labels.yml` file. Label colors are used to distinguish labels by category.
+
+Generally speaking, labels with the same category are considered mutually exclusive but in some cases labels sharing the same category can occur concurrently as they indicate qualifiers as opposed to types. For example, we may have the following types, https://github.com/conda/infrastructure/labels/type%3A%3Abug, https://github.com/conda/infrastructure/labels/type%3A%3Afeature, and https://github.com/conda/infrastructure/labels/type%3A%3Adocumentation, where for any one issue there would be _at most_ **one** of these to be defined (_i.e._ an issue shouldn’t be a bug _and_ a feature request at the same time). Alternatively, with issues involving specific operating systems (_i.e._, https://github.com/conda/infrastructure/labels/os%3A%3Alinux, https://github.com/conda/infrastructure/labels/os%3A%3Amacos, and https://github.com/conda/infrastructure/labels/os%3A%3Awindows), an issue could be labeled with one or more depending on the system(s) the issue is occurring on.
+
+Please note that there are also automation policies in place. For example, if an issue is labeled as https://github.com/conda/infrastructure/labels/pending%3A%3Afeedback and https://github.com/conda/infrastructure/labels/unreproducible, that issue will be auto-closed after a month of inactivity.
+
+
+#### What labels are required for each issue?
+
+At minimum, both `type` and `source` labels should be specified on each issue before moving it from "`Sorting`" to "`Backlog`". All issues that are bugs should also be tagged with a `severity` label.
+
+The `type` labels are exclusive of each other: each sorted issue should have exactly one `type` label. These labels give high-level information on the issue's classification (e.g., bug, feature, tech debt, etc.)
+
+The `source` labels are exclusive of each other: each sorted issue should have exactly one `source` label. These labels give information on the sub-group to which the issue's author belongs (e.g., a partner, a frequent contributor, the wider community). Through these labels, the conda organization hopes to gain insight into how well we're meeting the needs of various groups.
+
+The `severity` labels are exclusive of each other and should only be used when the `type::bug` label is also applied. These labels help us to prioritize our work. Severity is not the only factor for work prioritization, but it is an important consideration.
```

Comment:
Let's not restrict this to `type::bug`. I think we could easily add severity labels to documentation and feature request. In this case, the would describe how adversely the user is affected by the absence of something (e.g. if I had this feature, I wouldn't have to always use this annoying work-around solution).

Reply:
So `severity::*` is required for `type::bug` but can optionally be used elsewhere?

We may need to update the severity label descriptions to outline this since they are currently written with only bugs in mind